apps/web/.env.development

```diff
@@ -31,7 +31,7 @@ APPEALS_CASE_OFFICER_GROUP_ID=appeals_case_officer
 APPEALS_INSPECTOR_GROUP_ID=appeals_inspector
 APPEALS_VALIDATION_OFFICER_GROUP_ID=appeals_validation_officer
 APPLICATIONS_CASE_ADMIN_OFFICER_GROUP_ID=applications_case_admin_officer
-APPLICATIONS_CASEOFFICER_GROUP_ID=applications_case_officer
+APPLICATIONS_CASETEAM_GROUP_ID=applications_case_team
```

Comment:
All the env vars need to be updated in Azure and the infrastructure as well before this can be merged

Reply:
env vars updated: https://portal.azure.com/#@planninginspectorate.gov.uk/resource/subscriptions/962e477c-0f3b-4372-97fc-a198a58e259e/resourceGroups/pins-rg-back-office-dev-ukw-001/providers/Microsoft.Web/sites/pins-app-back-office-wfe-dev-ukw-001/configuration